rfc/text/advanced/ap_50_payload_passthru_mode.md

```diff
@@ -0,0 +1,401 @@
+        },
+        "com.myapp.secret_rpc_for_sensitive_data",
+        [Payload|binary]
+    ]
+```
+
+*Example.* Caller-to-Dealer progressive `CALL` with encryption and key ID.
+
+Nothing prevents to use `Payload Passthru Mode` with other features, for example `Progressive Calls`
+
+{align="left"}
+```json
+    [
+        48,
+        25471,
+        {
+            "ppt_scheme": "wamp.eth",
+            "ppt_serializer": "flatbuffers",
+            "ppt_cipher": "xsalsa20poly1305",
+            "ppt_keyid": "GTtQ37XGJO2O4R8Dvx4AUo8pe61D9evIWpKGQAPdOh0=",
+            "progress": true
+        },
+        "com.myapp.progressive_rpc_for_sensitive_data",
+        [Payload|binary]
+    ]
+```
+
+*Example.* Caller-to-Dealer `CALL` with mqtt payload. Specifying `"ppt_serializer": "mqtt"` means, that 
+original mqtt message payload is passed as WAMP payload message without any transcoding, as is.
+
+{align="left"}
+```json
+    [
+        48,
+        25471,
+        {
+            "ppt_scheme": "mqtt",
+            "ppt_serializer": "mqtt"
+        },
+        "com.myapp.mqtt_processing",
+        [Payload|binary]
+    ]
+```
+
+*Example.* Caller-to-Dealer `CALL` with mqtt payload. Specifying `"ppt_scheme": "mqtt"` simply indicates, that
+original source of payload data is received from related system. Specifying `"ppt_serializer": "json"` means, that
+original mqtt message payload was parsed and encoded with `json` serializer.
+
+{align="left"}
+```json
+    [
+        48,
+        25471,
+        {
+            "ppt_scheme": "mqtt",
+            "ppt_serializer": "json"
+        },
+        "com.myapp.mqtt_processing",
+        [Payload|binary]
+    ]
+```
+
+*Example.* Dealer-to-Callee `INVOCATION` with encryption and key ID
+
+{align="left"}
+```json
+    [
+        68,
+        35477,
+        1147,
+        {
+            "ppt_scheme": "wamp.eth",
+            "ppt_serializer": "cbor",
+            "ppt_cipher": "xsalsa20poly1305",
+            "ppt_keyid": "GTtQ37XGJO2O4R8Dvx4AUo8pe61D9evIWpKGQAPdOh0="
+        },
+        [Payload|binary]
+    ]
+```
+
+*Example.* Dealer-to-Callee `INVOCATION` with mqtt payload
+
+{align="left"}
+```json
+    [
+        68,
+        35479,
+        3344,
+        {
+            "ppt_scheme": "mqtt",
+            "ppt_serializer": "mqtt"
+        },
+        [Payload|binary]
+    ]
+```
+
+*Example.* Callee-to-Dealer `YIELD` with encryption and key ID
+
+{align="left"}
+```json
+    [
+        70,
+        87683,
+        {
+            "ppt_scheme": "wamp.eth",
+            "ppt_serializer": "flatbuffers",
+            "ppt_cipher": "xsalsa20poly1305",
+            "ppt_keyid": "GTtQ37XGJO2O4R8Dvx4AUo8pe61D9evNSpGMDQWdOh1="
+        },
+        [Payload|binary]
+    ]
+```
+
+*Example.* Callee-to-Dealer progressive `YIELD` with encryption and key ID
+
+Nothing prevents to use `Payload Passthru Mode` with other features, for example `Progressive Call Results`
```

Comment:
```suggestion
Nothing prevents the use of `Payload Passthru Mode` with other features such as, for example, `Progressive Call Results`
```

Reply:
This is sentence is repeated multiple times through the examples. It should be consolidated in one place.